packages/webcomponents/src/api/Checkout.ts

```diff
@@ -38,7 +38,8 @@ export interface ICheckout {
   bnpl?: IBnpl;
   total_amount: number;
   insurance_amount: number;
-};
+  status?: string;
```

Comment:
Can we do a union type of the actual values from the docs please? https://docs.justifi.tech/api-spec#tag/Checkouts/operation/CreateCheckout

`'created' | 'completed' | 'attempted' | 'expired'`